app/views/protocols/form/study_form_sections/_research_involving.html.haml

```diff
@@ -29,7 +29,7 @@
       .form-group.row.human_subjects
         = rt_form.label :human_subjects, t(:protocols)[:studies][:research_involving][:humans][:header], class: 'col-lg-2 control-label'
         .col-lg-1
-          = rt_form.check_box :human_subjects, class: 'form-control'
+          = rt_form.check_box :human_subjects, class: 'form-control human-subjects'
```

Comment:
Needs to be replicated in "app/views/dashboard/protocols/form/study_form_sections/_research_involving.html.haml"